Helper/Data.php

```diff
@@ -441,4 +525,18 @@ public function getStoreCodeToUrl(StoreInterface $store = null)
     {
         return (bool)$this->getStoreConfig(self::XML_PATH_STORE_CODE_TO_URL, $store);
     }
+
+    public function clearMagentoCache($type)
```

Comment:
Should we add a try catch block?

Reply:
DocBlock also missing

Reply:
@chenliangde I don't think anything inside `$this->cacheManager->clean()` throws an exception